landlab/components/landslides/landslide_probability.py

```diff
@@ -244,23 +257,23 @@ class property.
             "doc": "number of times FS is <=1 out of number of iterations user selected",
         },
         "soil__density": {
-            "dtype": float,
+            "dtype": int,
             "intent": "in",
```

Comment:
should soil density and soil internal friction angle really be int type? ditto the two fields below. These seem to me to be floats.